nucypher/blockchain/eth/sol/source/contracts/MiningAdjudicator.sol

```diff
@@ -144,7 +145,9 @@ contract MiningAdjudicator is Upgradeable {
         // Extract miner's address and check that is real miner
         // TODO: This will depend on the outcome of #962
         address miner = SignatureVerifier.recover(
-            SignatureVerifier.hash(_minerPublicKey, hashAlgorithm), _minerPublicKeySignature);
+            SignatureVerifier.hash(abi.encodePacked(precomp.lostBytes[4], miner_xcoord),
```

Comment:
...4?

Reply:
See https://github.com/nucypher/nucypher/blob/c86d0f38cc419fbc3528b97eb5ce9891dc8f399d/nucypher/blockchain/eth/sol/source/contracts/lib/UmbralDeserializer.sol#L217

BTW, there's a typo in the linked comment, and it should be 4 where it says 5. I'll correct it.